drivers/SlowCRC.h

```diff
@@ -0,0 +1,109 @@
+/* mbed Microcontroller Library
```

Comment:
Why do we have that if it's 'slow'? Does it have any advantages on 'fast'?

Reply:
Slow CRC does not create lookup table (less of RAM/ROM space), hence can be used by devices with limited memory.

Reply:
Can we call it small crc? And document these differences somewhere? I would say we could add it to file doxy and handbook later when we have a page.

Reply:
Small CRC will be misleading (related to length of CRC polynomial). I can rename it to BitwiseCRC, if that is fine.

Reply:
Yes I will add it to handbook (with example), once this PR is ready to merge.